README.md

```diff
@@ -68,15 +70,15 @@ Details for each individual task can be found using `mix help template`,
 `mix help tmplate.hex` and so on.
 
 See `Mix.Tasks.Gen` (in project
-[:mix_generator](https://github.com/pragdave/mix_generator)) for details 
+[:mix_generator](https://github.com/pragdave/mix_generator)) for details
 of how to use these templates.
 
 
 ### License
 
-Apache 2.0. See LICENSE.md for details.
+Apache 2.0. See [LICENSE.md](LICENSE.md) for details.
 
 ### See also
 
-For information on writing your own templates, see the moduledoc for MixTemplate,
+For information on writing your own templates, see the moduledoc for [MixTemplates](lib/mix_templates.ex),
```

Comment:
Provided a link directly into the source code for *some* viewing capability from github.